consensus/src/messages/handlers.rs

```diff
@@ -6,12 +6,12 @@ use std::{cmp, sync::Arc};
 use parking_lot::RwLock;
 
 use nimiq_block::Block;
+use nimiq_block::BlockInclusionProof;
 #[cfg(feature = "full")]
 use nimiq_blockchain::{Blockchain, CHUNK_SIZE};
 use nimiq_blockchain_interface::{AbstractBlockchain, Direction};
 use nimiq_blockchain_proxy::BlockchainProxy;
 use nimiq_network_interface::request::Handle;
-#[cfg(feature = "full")]
```

Comment:
I think we need to leave this as it is and then add:
`#[cfg(feature = "full")]` in the handle below

Reply:
Fixed